bd-buffer/src/ring_buffer_test.rs

```diff
@@ -250,7 +250,7 @@ async fn test_ring_buffer_stats() {
   );
 
   // Write a bunch of entries that will overflow the small buffers
-  for _ in 0 .. 1000 {
+  for _ in 0 .. 5000 {
```

Comment:
this test seemed a bit flaky so i just made it a bit more aggressive to see if that makes a difference